### PR TITLE
button events added

### PR DESCRIPTION
**Pull request by aburgosu**
button events added, please check it!

### CODE REVIEW

File: src/main/java/CoffeeMaker.java [hunk 1]
```diff
@@ -17,20 +17,20 @@ public CoffeeMaker() {
 		System.out.println("Coffeemaker created");
 	}
 
-	public void startProcess() throws InterruptedException {
+	public void startProcess(int qtyCups) throws InterruptedException {
 		warmerPlate.placePot();
-		System.out.println("How many cups of coffee do you want?");
-		Scanner inputScan = new Scanner(System.in);
-		int qtyCups = inputScan.nextInt();
+		//System.out.println("How many cups of coffee do you want?");
```
Comment (TheYisusGithub) [inline src/main/java/CoffeeMaker.java:22]: I recommend delete the coment.

File: src/main/java/Main.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	public static void main(String[] args) throws InterruptedException {
 		CoffeeMaker markIV = new CoffeeMaker();
 
-		markIV.startProcess();
+		markIV.startProcess(4);
```
Comment (MelissaRoman06) [inline src/main/java/Main.java:9]: The quantity of cups should be selectioned from comboBox displayed in the frame.

**Review by MelissaRoman06 — COMMENTED** [review]
Quantity of cups should be selected from comboBox, but this issue is not critical at the moment.

File: src/main/java/CoffeeMaker.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/CoffeeMaker.java:22]: This comment is already deleted in new commit.

**Pull request merged**